### DISCUSSION TITLE
How to send an ansi encoded file

### DISCUSSION THREAD

**Discussion by smaktacular** [AnWeber/vscode-httpyac#78] [Q&A] [answered]
I tried to send a 7zip file, which is likely ANSI encoded (according to notepad++), using the < operator. The webservice could not properly decode it. 
Afterwards I tried postman and the same file could be uploaded. 

1. What would be the correct way to tell the extension about the ansi encoding? <@ansi does not work as far as I can tell...
2. What other options instead of latin1 (from the docs) are valid ?

**Comment by AnWeber** [comment, marked as answer]
The [NodeJS encodings](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings) are supported. `ansi` is not included, use `ascii` instead. However, when reading in using `<`, no encoding is actually needed. Here the [byte[]](https://github.com/AnWeber/httpyac/blob/main/src/actions/createReqeustBodyInterceptor.ts#L78) is processed directly.
